test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -761,3 +761,202 @@ Feature: Bind values from a secret referred in backing service resource
             """
             bla
             """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+          """
+        Then Service Binding "bind-provisioned-service-1" is ready
+        And jq ".status.secret" of Service Binding "bind-provisioned-service-1" should be changed to "provisioned-secret-1"
+        And Content of file "/bindings/bind-provisioned-service-1/username" in application pod is
+            """
+            foo
+            """
+        And Content of file "/bindings/bind-provisioned-service-1/password" in application pod is
+            """
+            bar
+            """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service with mappings
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              bindAsFiles: true
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+                id: sec
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+              mappings:
+                - name: username_with_password
+                  value: '{{ .username }}:{{ .password }}'
```

Comment:
@pedjak Please let me know referring to Secret resource values like this is correct.
This seems to be working, but not sure the semantics is right.

When I tried to use the prefix with `data` (e.g., `.sec.data.username`) it gives the base64 encoded value. Is that also expected?

Reply:
Yes, both things work as expected.